MANUAL.md

```diff
@@ -446,7 +450,7 @@ The following extensions are currently defined for GMake:
 Appendix A GNU Free Documentation License
 Version 1.3, 3 November 2008
 
-Copyright © 2000, 2001, 2002, 2007, 2008 Free Software Foundation, Inc.
+Copyright Â© 2000, 2001, 2002, 2007, 2008 Free Software Foundation, Inc.
```

Comment:
And this?

Reply:
Looks like things moved to UTF-8 and GitHub is gagging. I will change over.